dts/sparc/gr716a.dtsi

```diff
@@ -85,5 +86,27 @@
 			label = "UART_5";
 			status = "disabled";
 		};
+
+		gpio0: gpio@8030c000 {
+			gpio-controller;
+			label = "gpio0";
+			#gpio-cells = <2>;
+			compatible = "gaisler,grgpio";
+			reg = <0x8030c000 0x1000>;
+			status = "disabled";
+			/* "Interrupt Line" 17, 18, 19, 20 */
+			interrupts = <17>;
+		};
+
+		gpio1: gpio@8030d000 {
+			gpio-controller;
+			label = "gpio1";
+			#gpio-cells = <2>;
+			compatible = "gaisler,grgpio";
+			reg = <0x8030d000 0x1000>;
+			status = "disabled";
+			/* "Interrupt Line" 38, 39, 40, 41 */
+			interrupts = <16>;
```

Comment:
comment seems confusing compared to the value.

Reply:
Agreed. I have removed the comments. (The numbers are related to the interrupt controller priority/remap functionality which is currently not configured in Zephyr.)